src/components/SearchResultItem.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from "react";
+import { iiifUrl } from "../utils";
+import { AppContext } from "../contexts/AppContext";
+
+/**
+ * SearchResultItem
+ */
+function SearchResultItem(props) {
```

Comment:
Could we document the shape of the props here. e.g. using propTypes or a doc block.

Reply:
Entertained using JSDoc as I generally prefer it, but went with prop-types as it seems like it might have benefits with eslint and I don't really know how to describe the React contexts correctly in JSDoc